sam_bot_description/sam_bot_description.urdf

```diff
@@ -0,0 +1,386 @@
+<?xml version="1.0"?>
+<robot name="sam_bot" xmlns:xacro="http://ros.org/wiki/xacro">

```

Comment:
It looks like you added a whole new file, not editing / fixing the existing one in the description directory (https://github.com/ros-planning/navigation2_tutorials/tree/61039a7eabd8c19d2bc5d88c32ba4a11b36daac2/sam_bot_description/src/description). Please make the changes to the existing URDF